src/pages/index.js

```diff
@@ -47,13 +88,73 @@ function fillProfileForm() {
 /*---------------------------------------------------------------------------------------------------------*/
 /*                                                Event Handlers                                           */
 /*---------------------------------------------------------------------------------------------------------*/
-const userInfo = new UserInfo(".profile__title", ".profile__description");
+const userInfo = new UserInfo(
+  ".profile__title",
+  ".profile__description",
+  ".profile__image"
+);
+
+function handleProfileEditSubmit(userData) {
+  formProfileEditModal.setLoading(true);
+  api
+    .updateUserProfile(userData)
+    .then(() => {
+      userInfo.setUserInfo(userData.name, userData.about);
+      formProfileEditModal.close();
+    })
+    .catch((err) => {
+      console.error(err);
+    })
+
+    .finally(() => {
```

Comment:
The api request is implemented correctly. everything are on the right places

Reply:
Thank you!